urls/_2021.py

```diff
@@ -214,4 +224,201 @@
+    '2021.4/tutorials/quality-control/qc-mock-3-observed.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/quality-control/qc-mock-3-observed.qza',
+    '2021.4/tutorials/quality-control/query-seqs.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/quality-control/query-seqs.qza',
+    '2021.4/tutorials/quality-control/reference-seqs.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/quality-control/reference-seqs.qza',
+    '2021.4/tutorials/quality-control/query-table.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/quality-control/query-table.qza',
+    '2021.4/tutorials/read-joining/atacama-seqs.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/read-joining/atacama-seqs.qza',
+    '2021.4/tutorials/read-joining/fj-joined.zip':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/read-joining/fj-joined.zip',
+    '2021.4/tutorials/sample-classifier/atacama-table.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/sample-classifier/atacama-table.qza',
+    '2021.4/tutorials/sample-classifier/moving-pictures-table.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/sample-classifier/moving-pictures-table.qza',
+    '2021.4/tutorials/training-feature-classifiers/85_otu_taxonomy.txt':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/training-feature-classifiers/85_otu_taxonomy.txt',
+    '2021.4/tutorials/training-feature-classifiers/85_otus.fasta':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/training-feature-classifiers/85_otus.fasta',
+    '2021.4/tutorials/training-feature-classifiers/rep-seqs.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2021.4/tutorials/training-feature-classifiers/rep-seqs.qza',
+    '2021.4/tutorials/phylogeny/rep-seqs.qza':
+        'https://qiime2-data.s3-us-west-2.amazonaws.com/2021.4/tutorials/phylogeny/rep-seqs.qza',
+    '2021.4/tutorials/utilities/faith-pd-vector.qza':
+        'https://qiime2-data.s3-us-west-2.amazonaws.com/2021.4/tutorials/utilities/faith-pd-vector.qza',
+    '2021.4/tutorials/utilities/jaccard-pcoa.qza':
+        'https://qiime2-data.s3-us-west-2.amazonaws.com/2021.4/tutorials/utilities/jaccard-pcoa.qza',
+    '2021.4/tutorials/utilities/taxa-barplot.qzv':
+        'https://qiime2-data.s3-us-west-2.amazonaws.com/2021.4/tutorials/utilities/taxa-barplot.qzv',
+
+    # Sample Metadata (hosted on Google Sheets)
+
+    ## FMT
+    '2021.4/tutorials/fmt/sample_metadata':
+        'https://docs.google.com/spreadsheets/d/15fWkBz5O2JH5wK5-fXLMJR2ZP4-rj8lMMTFzCaRVxuU/edit?usp=sharing',
+    '2021.4/tutorials/fmt/sample_metadata.tsv':
+        'https://docs.google.com/spreadsheets/d/15fWkBz5O2JH5wK5-fXLMJR2ZP4-rj8lMMTFzCaRVxuU/export?gid=0&format=tsv',
```

Comment:
I did this one before I remembered the note in the Google Doc that you wanted to do this with Keegan, @thermokarst.

Reply:
Thanks! That note was actually for you, @andrewsanchez (all of those "let's do this together" notes were targeted at you), but it was just because I wanted to make sure you had gdrive access etc. Sounds like you did, so no worries.